src/server/models/match.model.js

```diff
@@ -0,0 +1,28 @@
+import mongoose from 'mongoose';
+const Schema = mongoose.Schema;
+
+let matchSchema = new Schema({
+    mentorId: Schema.Types.ObjectId,
+    menteeId: Schema.Types.ObjectId,
+    requestDate: {
+        type: Date,
+        default: Date.now(),
+    },
+    isAccepted: {
+        type: Boolean,
+        default: false,
```

Comment:
What decides whether to set default value or not?
The ones we currently have default values for make perfect sense. But things like mentorId or acceptDate / cancelDate having no default values makes me wonder if they should have default values too..